packages/drive/index.js

```diff
@@ -61,11 +64,18 @@ export class Drivestore {
    * Get a Hyperdrive by its name.
    */
   get (name = 'public') {
+    const opened = this._openDrives.get(name)
+    if (opened && !opened.core.closed) return opened
```

Comment:
Please, first validate input and then use it  (move below `validateName(name)` call

Reply:
There isn't really a problem if the name was invalid here, but good practice.